File: Sample/app/src/main/java/com/example/workspace/sample/util/FragmentUtil.java
package com.example.workspace.sample.util;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentTransaction;

public class FragmentUtil {
    private final FragmentManager fragmentManager;
    private final int containerViewId;

    public FragmentUtil(FragmentManager manager, int containerViewId) {
        this.fragmentManager = manager;
        this.containerViewId = containerViewId;
    }

    public void openFragment(Fragment fragment) {
        FragmentTransaction transaction = fragmentManager.beginTransaction();
        transaction.replace(containerViewId, fragment).addToBackStack(null).commit();
    }
}

File: Sample/app/src/main/java/com/example/workspace/sample/util/FragmentUtil.kt
package com.example.workspace.sample.util

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentTransaction

class FragmentUtil(private val fragmentManager: FragmentManager, private val containerViewId: Int) {

    fun openFragment(fragment: Fragment) {
        val transaction = fragmentManager.beginTransaction()
        transaction.replace(containerViewId, fragment).addToBackStack(null).commit()
    }
}
